### PR TITLE
Feature/271 lambert surface relative velocity

### PR DESCRIPTION
**Pull request by JulianHammerl**
* **Tickets addressed:** Confluence 271
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

A module was created that computes the inertial velocity required to achieve a desired velocity relative to the surface of a celestial body at the end of a Lambert problem transfer arc (at the second position vector of Lambert problem). The computed inertial velocity can then be used by the module in #109 to obtain the Delta-V that should be performed at the end of the Lambert problem transfer arc. The commits are organized as follows:

- Commit 1: Create necessary message definitions (from PR #109)
- Commit 2: Create module
- Commit 3: UnitTest
- Commit 4: Documentation

## Verification

A UnitTest was created that compares the computed Delta-V with the true value.

## Documentation

This is a new module. An RST documentation was added.

## Future work

None at this point.

### CODE REVIEW

File: src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+angularVelocities = np.array([[0., 0., 0.], [1., -3., 2.]])
+
+paramArray = [relativeVelocities, time_maneuver, trueAnomalies, rotAngles, angularVelocities]
+# create list with all combinations of parameters
+paramList = list(itertools.product(*paramArray))
+
+@pytest.mark.parametrize("accuracy", [1e-4])
+@pytest.mark.parametrize("p1_vr, p2_tm, p3_f, p4_rot, p5_omega", paramList)
+
+def test_lambertValidator(show_plots, p1_vr, p2_tm, p3_f, p4_rot, p5_omega, accuracy):
```
Comment (thibaudteil) [inline src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py]: Name is from a previous module

File: src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        :param p4_rot: rotation angle of planet
+        :param p5_omega: angular velocity vector of planet
+        :param accuracy: accuracy of the test
+
+    **Description of Variables Being Tested**
+
+    The content of the DesiredVelocityMsg output message (velocity vector and maneuver time) is compared with the true
+    values.
+    """
+    lambertValidatorTestFunction(show_plots, p1_vr, p2_tm, p3_f, p4_rot, p5_omega, accuracy)
```
Comment (thibaudteil) [inline src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py]: Same here

File: src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py (hunk 1 above)
Comment (JulianHammerl) [reply src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py]: Fixed.

File: src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py (hunk 2 above)
Comment (JulianHammerl) [reply src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py]: Fixed and slightly embarrassed.

**Review by thibaudteil — APPROVED** [review]

**Pull request merged**